app/views/home/feed.xml.builder

```diff
@@ -0,0 +1,16 @@
+atom_feed do |feed|
```

Comment:
Since this lives under the `:locale` piece of the routes, we should probably pass `:language` in to `atom_feed` and pass in the current locale.